assets/javascripts/discourse/routes/admin-plugins-discourse-subscriptions-subscriptions.js.es6

```diff
@@ -1,36 +1,29 @@
+import I18n from "I18n";
 import Route from "@ember/routing/route";
 import AdminSubscription from "discourse/plugins/discourse-subscriptions/discourse/models/admin-subscription";
-import I18n from "I18n";
 
 export default Route.extend({
   model() {
     return AdminSubscription.find();
   },
 
   actions: {
-    cancelSubscription(subscription) {
-      bootbox.confirm(
-        I18n.t(
-          "discourse_subscriptions.user.subscriptions.operations.destroy.confirm"
-        ),
-        I18n.t("no_value"),
-        I18n.t("yes_value"),
-        (confirmed) => {
-          if (confirmed) {
-            subscription.set("loading", true);
-
-            subscription
-              .destroy()
-              .then((result) => subscription.set("status", result.status))
-              .catch((data) =>
-                bootbox.alert(data.jqXHR.responseJSON.errors.join("\n"))
-              )
-              .finally(() => {
-                subscription.set("loading", false);
-              });
-          }
-        }
-      );
+    cancelSubscription(model) {
+      const subscription = model.subscription;
+      const refund = model.refund;
+      subscription.set("loading", true);
+      subscription
+        .destroy(refund)
+        .then((result) => subscription.set("status", result.status))
+        .catch((data) =>
+          bootbox.alert(data.jqXHR.responseJSON.errors.join("\n"))
+        )
+        .finally(() => {
+          this.send("closeModal");
+          bootbox.alert(I18n.t("discourse_subscriptions.admin.canceled"));
```

Comment:
`bootbox#alert` will be called twitce if the request fails. Maybe this should be moved inside the `then` callback?

Reply:
Good call. Fixing that.